packages/headless/src/controllers/facets/range-facet/date-facet/headless-date-facet-options.ts

```diff
@@ -105,3 +113,21 @@ export const dateFacetOptionsSchema = new Schema<Required<DateFacetOptions>>({
   }),
   sortCriteria: new StringValue({constrainTo: rangeFacetSortCriteria}),
 });
+
+export function validateManualDateRanges(options: DateFacetOptions) {
+  options.currentValues?.forEach((value) => {
+    if (dayjs(value.start).isAfter(dayjs(value.end))) {
```

Comment:
I'm guessing there's also the possibility that `dayjs(value.start)` itself would throw an error, if the value is not a date at all.

Should there be a test for this ? To verify what behaviour happens ?